waitress/adjustments.py

```diff
@@ -66,6 +66,11 @@ def slash_fixed_str(s):
         s = '/' + s.lstrip('/').rstrip('/')
     return s
 
+def str_ifnotnone(s):
+    if s is None:
+        return None
```

Comment:
This line is not yet covered by the test. The test is using a bunch of dummies, instead you'll want to add to https://github.com/Pylons/waitress/blob/master/waitress/tests/test_adjustments.py